kubeprod/cmd/install.go

```diff
@@ -96,7 +96,7 @@ func NewInstallSubcommand(cmd *cobra.Command, platform string, config installer.
 
 	c.Config, err = clientConfig.ClientConfig()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Unable to read kubectl config: %v", err)
```

Comment:
golang errors should start with a lowercase.  I keep forgetting this, which is mostly why I'm making a fuss about it now ;)

Reply:
fixed!